site/v5/professional/professional-setup.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Set up
+date: 2014-04-18 11:30:00 
+permalink: /v5/professional/setup.html
+eleventyNavigation:
+  version: v5
+  order: 1
+---
+<h1 class="title">Professional - Set up</h1>
+
+[[TOC]]
+
+# Introduction
+This sections explains and goes through all the steps to get up and running the Mongock professional
+
+
+# Steps
+
+### 1. Request a license key
+
+[This link](https://www.mongock.io/download) will take you to the Mongock website, where you just need to provide some basic information, such as the email, where you will receive the license key
```

Comment:
I'd change it to something like:

You can request a License key from the following link (link). The website will display a form which will send a license key to the desired email address.